### ISSUE TITLE
[Vote] Support Chrome-based browsers

### ISSUE THREAD

**Issue by ilovefreesw** [wrbl606/what-to-click#4]
Would be nice to have it on Chrome and Microsoft Edge.

Necessary steps:
- [x] Make the extension work with offline code only
- [ ] Enable Manifest 3: Adjust `manifest.json`
- [ ] Enable Manifest 3: Update browser extension code
- [ ] Chromium compatibilty: Use APIs properly supported by both Firefox and Chromium

**Comment by wrbl606** [comment]
Thanks for your suggestion ❤️
There are significant changes between Firefox's and Chrome's browser extension APIs which makes it difficult to maintain code for both targets. 

For anybody reading this, that would also be interested in Chrome version of What-to-click, please leave a reaction to the original message or comment so I can see how big the demand is.

**Comment by m3thm4th** [comment]
I don't think that would be a valid representation of the general interest, [Scribe](https://chrome.google.com/webstore/detail/scribe-%E2%80%94-documentation-so/okfkdaglfjjjfefdcppliegebpoegaii) has 400,000+ users at the time of writing this, I'm sure if they had the choice of a private alternative they would be really happy. So I would assume the interest is there, but first you should perfect Firefox's version by fixing #1 and make this extension work completely offline, that will be your biggest "selling point".

**Comment by wrbl606** [comment]
Necessary steps:
- [x] Make the extension work with offline code only
- [ ] Enable Manifest 3: Adjust `manifest.json`
- [ ] Enable Manifest 3: Update browser extension code
- [ ] Chromium compatibilty: Use APIs properly supported by both Firefox and Chromium

**Comment by PathanTech** [comment]
Would love to see this in Chrome browser too,
Kindly launch
